src/main/java/org/sagebionetworks/web/client/widget/entity/menu/v3/EntityActionMenuViewImpl.java

```diff
@@ -95,21 +67,6 @@ private void renderLoaderComponent() {
     loaderComponent.render(node);
   }
 
-  private EntityActionMenuProps getProps() {
```

Comment:
Logic was moved to the EntityActionMenuProps POJO (`EntityActionMenuProps.toJsInterop`)